src/tests/graphql/explanationOfBenefit/graphql.explanationOfBenefits.test.js

```diff
@@ -38,117 +38,99 @@ describe('GraphQL ExplanationOfBenefit Tests', () => {
-                        });
-                        expect(body.data.explanationOfBenefit).toStrictEqual(expected);
-                    }, cb),
-            ]);
+            let resp = await request
+                .get('/4_0_0/ExplanationOfBenefit')
+                .set(getHeaders())
+                .expect(200);
+            expect(resp.body.length).toBe(0);
+            console.log('------- response 1 ------------');
+            console.log(JSON.stringify(resp.body, null, 2));
+            console.log('------- end response 1 ------------');
+
+            resp = await request
+                .post('/4_0_0/Patient/1/$merge')
+                .send(patientBundleResource)
+                .set(getHeaders())
+                .expect(200);
+            console.log('------- response 2 ------------');
+            console.log(JSON.stringify(resp.body, null, 2));
+            console.log('------- end response 2  ------------');
+
+            resp = await request
+                .post('/4_0_0/Organization/1/$merge')
+                .send(organizationBundleResource)
+                .set(getHeaders())
+                .expect(200);
+            console.log('------- response 2 ------------');
+            console.log(JSON.stringify(resp.body, null, 2));
+            console.log('------- end response 2  ------------');
+
+            resp = await request
+                .post('/4_0_0/Coverage/1/$merge')
+                .send(coverageBundleResource)
+                .set(getHeaders())
+                .expect(200);
+            console.log('------- response 2 ------------');
+            console.log(JSON.stringify(resp.body, null, 2));
+            console.log('------- end response 2  ------------');
+
+            resp = await request
+                .post('/4_0_0/ExplanationOfBenefit/1/$merge')
+                .send(explanationOfBenefitBundleResource)
+                .set(getHeaders())
+                .expect(200);
+            console.log('------- response 2 ------------');
+            console.log(JSON.stringify(resp.body, null, 2));
+            console.log('------- end response 2  ------------');
+
+            resp = await request
+                .get('/4_0_0/Patient/')
+                .set(getHeaders())
+                .expect(200);
+            console.log('------- response patient ------------');
+            console.log(JSON.stringify(resp.body, null, 2));
+            console.log('------- end response patient  ------------');
+
+            resp = await request
+                .get('/4_0_0/ExplanationOfBenefit/')
+                .set(getHeaders())
+                .expect(200);
+            console.log('------- response 2 ------------');
+            console.log(JSON.stringify(resp.body, null, 2));
+            console.log('------- end response 2  ------------');
+
+            resp = await request
+                // .get('/graphql/?query=' + graphqlQueryText)
+                // .set(getHeaders())
```

Comment:
Is this commented out to be added back in later?

Reply:
just to remind folks there are two ways to get graphql